src/pages/messages/components/PublishingRequestMessagesColumn.tsx

```diff
@@ -16,7 +16,11 @@ export const StyledStatusMessageBox = styled(Box)({
   borderRadius: '4px',
 });
 
-export const PublishingRequestMessagesColumn = ({ ticket }: { ticket: ExpandedPublishingTicket }) => {
+export const PublishingRequestMessagesColumn = ({
+  ticket,
+}: {
+  ticket: ExpandedPublishingTicket | PublishingTicket;
+}) => {
```

Comment:
Kanskje mer lesbart å legge props i `interface PublishingRequestMessagesColumnProps` for å unngå at dette blir over 5 linjer